WanderLost/WanderLost/Client/Components/Card.razor

```diff
@@ -1,26 +1,80 @@
-﻿<div class="card card_r--@(Merchant.Card.Rarity) @(ClassName)">
+﻿@inject MerchantHubClient HubClient
+
+<div class="card card_r--@(TopRarity) @(ClassName)">
     <div class="card__title">@(Merchant.Name)&nbsp;(@(Merchant.Zone))</div>
     <div class="card__location" style="background-image: url(/images/zones/@(Uri.EscapeDataString(Merchant.Zone)).jpg)" />
     <div class="stock">
         <div class="stock__title">Inventory:</div>
-        <div class="stock__item stock__item_r--@(Merchant.Card.Rarity)">
+        <div class="stock__item stock__item_r--@(CardRarity)">
             @(Merchant.Card.Name)
         </div>
-        <div class="stock__item stock__item_r--@(Merchant.Rapport.Rarity)">
+        <div class="stock__item stock__item_r--@(RapportRarity)">
             @(Merchant.Rapport.Name)
         </div>
     </div>
     <div class="card__buttons">
-        <CardButton Type="CardButton.Variant.Like" Active=true />
-        <div class="card__score">0</div>
-        <CardButton Type="CardButton.Variant.Dislike" Active=false />
+        <CardButton Type="CardButton.Variant.Like" OnClick="@(async () => await SubmitVote(VoteType.Upvote))" Active=true />
```

Comment:
you need to set `Active` flag based on the user vote for this card

Reply:
My bad, forgot about that, will change